docker/Dockerfile

```diff
@@ -13,21 +13,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:18.04
+FROM alpine:3.13
 MAINTAINER Apache Nutch Committers <dev@nutch.apache.org>
 
 WORKDIR /root/
 
-
 # Install dependencies
-RUN apt update
-RUN apt install -y ant git openjdk-8-jdk-headless
-
-# Set up JAVA_HOME
-RUN echo 'export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64' >> $HOME/.bashrc
```

Comment:
JAVA_HOME needs to be set
```
RUN echo 'export JAVA_HOME=/usr/lib/jvm/java-11-openjdk' >> $HOME/.bashrc
```
otherwise bin/nutch fails:
```
bash-5.1# bin/nutch parsechecker https://nutch.apache.org/
Error: JAVA_HOME is not set.
```